### ISSUE TITLE
Using onset for cross-domain Reid

### ISSUE THREAD

**Issue by justopit** [KaiyangZhou/deep-person-reid#299]
Hi! I noticed that when I just use PCB, **resnet50_ibn_b**, they does't provide good performance. However, your onset_ibn_b construct a strong backbone for cross-domain Reid. What is, you think ,  the key of the success of the strong backbone?

**Comment by KaiyangZhou** [comment]
briefly, the key is to learn features that are both discriminative and generalisable at the same time, that's why we propose omni-scale feature learning and NAS+instance normalisation to solve this problem

See our latest paper which discusses the cross-domain issue in re-id https://arxiv.org/abs/1910.06827

**Comment by justopit** [comment]
Thank you. By the way, using color jitter and label smoothing, the cross-reed performance of model PCB is similar to your onset_ibn. However, it's a lot worse than your osnet_ain.

**Comment by ft3020997** [comment]
> briefly, the key is to learn features that are both discriminative and generalisable at the same time, that's why we propose omni-scale feature learning and NAS+instance normalisation to solve this problem
> 
> See our latest paper which discusses the cross-domain issue in re-id https://arxiv.org/abs/1910.06827

Hi ! 
 I had read your paper about cross-domain test on MSMT17-MARKET1501 (result: R1-71.1 mAP-52.7), but what I have tested by the torchreid config file of this (im_osnet_ain_x1_0_softmax_256x128_amsgrad_cosine.yaml) did not get that result.
So did I miss any config or tricks on training?
My Env info:
pytorch: 1.3.1
torchreid: 1.0.9

Best Wish.

**Comment by KaiyangZhou** [comment]
@ft3020997 
please provide your full command and report your results

**Comment by ft3020997** [comment]
> @ft3020997
> please provide your full command and report your results

Thanks for response.
Full running command: 
python scripts/main.py  --config-file im_osnet_ain_x1_0_softmax_256x128_amsgrad_cosine.yaml --root $DATA$ 
(all configs are almost same with default yaml file, besides sources and targets, and default_config.py has never been modified)

The results as showed:
** Results **
mAP: 31.3%
CMC curve
Rank-1  : 60.5%
Rank-5  : 77.3%
Rank-10 : 83.1%
Rank-20 : 87.8%
Checkpoint saved to "log/osnet_ain_x1_0_msmt17_market1501_softmax_cosinelr/model.pth.tar-100"

GPU envs:
CUDA:10.0 (v100)

**Comment by KaiyangZhou** [comment]
@ft3020997 
set `combineall: True` for msmt17 (please refer to the paper for details)
